### PR TITLE
Sriov hybrid templates 

### PR DESCRIPTION
**Pull request by pinikomarov**

### CODE REVIEW

**Comment by pinikomarov** [comment]
This feature is turned on by : 
- export OSP_RELEASE=16.2_fencing_3combo_sriov
- cp  ansible/vars/local_templates_hybrid_sriov.yaml -> ansible/local-defaults.yaml

File: ansible/vars/16.2_fencing_3combo_sriov.yaml [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+  - ctlplane
+  - external
+  - internal_api
+
+enable_fencing: true
+
+ocp_num_workers: 0
+ocp_master_memory: 80000
+ocp_master_vcpu: 24
+ocp_master_disk: 100
```
Comment (abays) [inline ansible/vars/16.2_fencing_3combo_sriov.yaml:12]: I think we should lower the CPU and memory values across the various hybrid configurations.  I know I originally provided these values in my hybrid document, but I was using them on an extremely powerful machine.  Perhaps `12` for CPU and `40000` for memory would be a more reasonable standard.

File: ansible/vars/local_templates_hybrid_sriov.yaml [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+# 3 virtualized master/worker combo OCP nodes
+# No dedicated virtual workers
+# No virtual OSP computes
+ocp_num_masters: 3
+ocp_num_workers: 0
+ocp_num_extra_workers: 0
+
+# Extra specs for the master/workers combo nodes
+ocp_master_memory: 80000
+ocp_master_vcpu: 24
```
Comment (abays) [inline ansible/vars/local_templates_hybrid_sriov.yaml:30]: Same comment here as before regarding CPU/memory specs

File: ansible/templates/osp/tripleo_tarball_config/16.2/roles_data.yaml.j2 [hunk 3]
```diff
@@ -851,3 +851,253 @@
...
+    - OS::TripleO::Services::Timesync
+    - OS::TripleO::Services::Timezone
+    - OS::TripleO::Services::TripleoFirewall
+    - OS::TripleO::Services::TripleoPackages
+    - OS::TripleO::Services::Vpp
+    - OS::TripleO::Services::OVNController
+    - OS::TripleO::Services::OVNMetadataAgent
+###############################################################################
+# Role: ControllerSriov                                                       #
+###############################################################################
```
Comment (abays) [inline ansible/templates/osp/tripleo_tarball_config/16.2/roles_data.yaml.j2:921]: Did the out-of-the-box `Controller` role not suffice for this deployment?  Were there extra `ServicesDefault` that you had to add, and thus you made this custom role?

File: ansible/vars/16.2_fencing_3combo_sriov.yaml (hunk 1 above)
Comment (pinikomarov) [reply ansible/vars/16.2_fencing_3combo_sriov.yaml]: ok updated the file with those numbers

Comment (pinikomarov) [reply ansible/vars/16.2_fencing_3combo_sriov.yaml]: ok updated those numbers

File: ansible/vars/local_templates_hybrid_sriov.yaml (hunk 2 above)
Comment (pinikomarov) [reply ansible/vars/local_templates_hybrid_sriov.yaml]: ok updated those numbers

File: ansible/templates/osp/tripleo_tarball_config/16.2/roles_data.yaml.j2 (hunk 3 above)
Comment (pinikomarov) [reply ansible/templates/osp/tripleo_tarball_config/16.2/roles_data.yaml.j2:921]: the ControllerSriov has an extra : 
    - OS::TripleO::Services::OVNMetadataAgent
that we need

File: ansible/vars/local_templates_hybrid_sriov.yaml [hunk 4]
```diff
@@ -0,0 +1,73 @@
...
+    bmc_address: 10.46.2.224
+    bmc_username: root
+    bmc_password: calvin
+  worker-1:
+    vendor: Dell
+    prov_mac: 6c:fe:54:0a:bc:f2 # using ens1f2
+    bmc_protocol: idrac         
+    bmc_address: 10.46.2.226
+    bmc_username: root
+    bmc_password: calvin
```
Comment (abays) [inline ansible/vars/local_templates_hybrid_sriov.yaml:73]: Same question here I asked in the SRIOV PR: do we feel comfortable with leaving these creds visible to the public?

File: ansible/vars/local_templates_hybrid_sriov.yaml (hunk 2 above)
Comment (pinikomarov) [reply ansible/vars/local_templates_hybrid_sriov.yaml]: done

**Review by abays — COMMENTED** [review]
I think this is good to go once we remove the private creds

**Comment by pinikomarov** [comment]
ok moving creds to : https://gitlab.cee.redhat.com/openstack-k8s-operators/dev-tools/-/merge_requests/2

**Pull request merged**